pages/event.php

```diff
@@ -170,17 +172,54 @@ function event() {
             // Change the item ID values based on dev/prod
             // These are placeholder exchange options
             switch ($_GET['exchange']) {
-                case "red_yen":
+                case "red_yen_small":
                     $player->getInventory();
                     if (!$player->hasItem($system->event_data['red_lantern_id'])) {
                         throw new RuntimeException("You do not have this item!");
                     }
-                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < 3) {
+                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < 1) {
                         throw new RuntimeException("You do not have enough of this item!");
                     }
-                    $player->items[$system->event_data['red_lantern_id']]->quantity -= 3;
-                    $player->addMoney("1000", "Event");
-                    $system->message("You have traded 3 Red Lanterns for 1000 yen!");
+                    $player->items[$system->event_data['red_lantern_id']]->quantity -= 1;
+                    if ($player->items[$system->event_data['red_lantern_id']]->quantity < 1) {
+                        unset($player->items[$system->event_data['red_lantern_id']]);
+                    }
+                    $player->addMoney("25", "Event");
```

Comment:
```suggestion
                    $player->addMoney(25, "Lantern Event");
```

Reply:
Hmm actually would be nice if this all scaled off a single var like `$yen_per_lantern` so we don't have to update multiple numbers if we want to adjust the amount